### DISCUSSION TITLE
Parallel Computing

### DISCUSSION THREAD

**Discussion by Jagannath009** [STEllAR-GROUP/hpx#6011] [General]
Hello everyone  

I'm Jagannath kumar, a computer science student and software engineer intern in India.I did my Postgraduation from BITS PILANI GOA  I'm also a competitive programmer.

I'm interested in the Stellar organization; I came across your website and find really interesting projects you've worked on. 

I'm surprised by the variety of those projects. That's why I want to take part in that great community. I'm willing to join you as part of the GSOC 2022 program and after that also. As a software engineer, I leveraged a lot of open-source tools, so that'll be my way of giving back to the community.

I'm more interested in back-end engineering with an experience with the front end.

I will try harder to work with the community to deliver good products!

Thank you[🙂

**Comment by hkaiser** [comment]
@chapaith this is an open-source project. You don't have to ask for permission to start working. Familiarize yourself with the code, examples, issues, etc. and pick something that's interesting to you. Let us know what you chose to work on so we can coordinate efforts.
